website/content/en/docs/getting-started/_index.md

```diff
@@ -0,0 +1,236 @@
+(permission sets) on AWS. 
+
+First, define a role using the template below. It provides full access to EC2,
+and limited access to other services such as EKS and Elastic Container Registry
+(ECR).
+
+```bash
+# Creates IAM resources used by Karpenter
+TEMPOUT=$(mktemp)
+curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/"${KARPENTER_VERSION}"/docs/aws/karpenter.cloudformation.yaml > $TEMPOUT \
+&& aws cloudformation deploy \
+  --stack-name Karpenter-${CLUSTER_NAME} \
+  --template-file ${TEMPOUT} \
+  --capabilities CAPABILITY_NAMED_IAM \
+  --parameter-overrides ClusterName=${CLUSTER_NAME}
+```
+
+Second, create the mapping between Kubernetes resources and the new IAM role. 
+
+```bash
+# Add the Karpenter node role to your aws-auth configmap, allowing nodes with this role to connect to the cluster.
+eksctl create iamidentitymapping \
+  --username system:node:{{EC2PrivateDNSName}} \
+  --cluster  ${CLUSTER_NAME} \
+  --arn arn:aws:iam::${AWS_ACCOUNT_ID}:role/KarpenterNodeRole-${CLUSTER_NAME} \
+  --group system:bootstrappers \
+  --group system:nodes
+```
+
+Now, Karpenter can send requests for new EC2 instances to AWS. 
+
+### Install Karpenter Helm Chart
+
+Use helm to deploy Karpenter to the cluster. 
+
+We created a Kubernetes service account when we created the cluster using
+eksctl. Thus, we don't need the helm chart to do that.
+
+```bash
+helm repo add karpenter https://awslabs.github.io/karpenter/charts
+helm repo update
+helm upgrade --install karpenter karpenter/karpenter \
+  --namespace karpenter --set serviceAccount.create=false
+```
+
+### Provisioner
+
+A single Karpenter provisioner is capable of handling many different pod
+shapes. In other words, Karpenter eliminates the need to manage many different
+node groups. Karpenter makes scheduling and provisioning decisions based on pod
+attributes such as labels and affinity. 
+
+Create a simple default provisioner using the command below. This provisioner
+provides instances with the default certificate bundle, and the control plane
+endpoint url. 
+
+Importantly, the `ttlSecondsAfterEmpty` value configures Karpenter to
+deprovision empty nodes. 
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: provisioning.karpenter.sh/v1alpha2
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  cluster:
+    name: ${CLUSTER_NAME}
+    caBundle: $(aws eks describe-cluster --name ${CLUSTER_NAME} --query "cluster.certificateAuthority.data" --output json)
+    endpoint: $(aws eks describe-cluster --name ${CLUSTER_NAME} --query "cluster.endpoint" --output json)
+  ttlSecondsAfterEmpty: 30
+EOF
+kubectl get provisioner default -o yaml
+```
+
+## First Use
+
+Karpenter is now active and ready to begin provisioning nodes. Create a
+workload (e.g., deployment) to see Karpenter provision some nodes. 
+
+### Create Workload
+
+This deployment uses the [pause image](https://www.ianlewis.org/en/almighty-pause-container) and starts with zero replicas. 
```

Comment:
We also have the feature to include a `karpenter.sh/do-not-evict:true` annotation for pods they don't want to evict. This is a pretty specific feature, so it may overcomplicate this guide. WDYT?

Reply:
i'll include that in the annotations section